### ISSUE TITLE
Discor.SteamOpenIdRedirectUri not being set?

### ISSUE THREAD

**Issue by smooreace** [ark-mod/ArkBot#97]
Finally finished adding 3 clusters of data and relaunching ArkBot to get the above error. 
Opening the config to look works
Expanding every other section works fine as well
Expanding the Discord section leads to a crash
[application.log](https://github.com/ark-mod/ArkBot/files/3503399/application.log)
[applicationcrash_CurrentDomain_UnhandledException_2019-08-14.14.25.07.8445.log](https://github.com/ark-mod/ArkBot/files/3503400/applicationcrash_CurrentDomain_UnhandledException_2019-08-14.14.25.07.8445.log)

**Comment by jefferson-1** [comment]
If you want to help sponsor a bugfix I might be able to get a dev to look at it... I've been sponsoring bugfixes on Gitcoin.co

**Comment by smooreace** [comment]
I disabled Discord bot function via json edit and got past that

**Comment by GildedHonour** [comment]
Hi. I want to discuss contribution to or working on this project, who can I discuss it with? In Telegram or Skype, perhaps 
@jefferson-1 cc @tsebring

**Comment by jefferson-1** [comment]
Yes please join discord and we can discuss. https://discord.gg/zDJcckF

**Comment by GildedHonour** [comment]
I've joined and dropped a message in the main chat

**Comment by jefferson-1** [comment]
Please join Arkbotusers ... react to get the role.

**Comment by GildedHonour** [comment]
Is that a channel? It doesn't exist.


El 26 de agosto de 2019 15:39:53 GMT+03:00, jefferson-1 <notifications@github.com> escribió:
>Please join Arkbotusers ... react to get the role.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/ark-mod/ArkBot/issues/97#issuecomment-524844089

-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

**Comment by GildedHonour** [comment]
I don't see "ArkBotDev" in the channel anywhere. Only 3 posts by Contance and nothing else. Nor am I allowed to post. What do you mean by "react"?

My discord: "GildedHonour#7713", text me

**Comment by GildedHonour** [comment]
I added you.

**Comment by jefferson-1** [comment]
Try to message me? I don't yet see you on my friends list.

**Comment by GildedHonour** [comment]
I'm confused. I clicked "accept" on your request. Then you disappeared somewhere. What's you discord ID? Or let's do that on skype this time, it'll be faster -- "gilded.honour"

**Comment by tsebring** [comment]
This error happens if you write something like this in the config
`    "enabledChannels": [
      null
    ],`

Exception is thrown deep within Xceed.Wpf.Toolkit so the main thing we can do to avoid this problem is to validate the config looking specifically for things that are known to break the configuration editor.
